app/assets/stylesheets/mobile/comments.scss

```diff
@@ -8,44 +8,100 @@
   margin-top: 10px;
   border-top: 1px solid $border-grey;
   min-height: 22px;
+  overflow: hidden;
 
-  > a, .show_comments {
+  > a,
+  .show-comments,
+  .show-comments > [class^="entypo"] {
     @include transition(color);
     color: $text-grey;
     font-weight: bold;
   }
 
-  .show_comments {
+  .show-comments {
     position: relative;
     top: 3px;
-  }
 
-  a.show_comments.active:not(.bottom_collapse) {
-    color: $text;
-    padding-right: 14px;
-    background: {
-      image: image-url("mobile/arrow_down_small.png");
-      position: center right;
-      repeat: no-repeat;
+    &:hover,
+    &:active,
+    &:focus {
+      outline: none;
+      text-decoration: none;
+    }
+
+    &.active:not(.bottom_collapse),
+    &.active:not(.bottom_collapse) > [class^="entypo"] {
+      color: $text;
     }
   }
 
-  .floater {
+  .post-stats {
     top: -5px;
     float: right;
-    height: 28px;
-    overflow: hidden;
     position: relative;
+    display: flex;
+
+    .comment-button:focus { outline: none; }
```

Comment:
Since we're doing nice accessibility improvements on the desktop version, it feels wrong to ignore the mobile version. Having a distinguishable focus effect may be important for some users (even me when using a blackberry with a physical keyboard and using the keyboard touch capabilities to scroll around the page), so I think removing all focus effects is wrong.

I agree that the outline does not look nice all the time, but after all, it's a nice standard. If you feel like removing it, I think there should be another indicator.

What do you think?